lib/matplotlib/text.py

```diff
@@ -2398,6 +2415,26 @@ def get_window_extent(self, renderer=None):
 
         return Bbox.union(bboxes)
 
+    def _set_to_best_position(self, renderer):
+        if self.axes is None:
+            return
+        bbox, info, descent = self._get_layout(renderer)
+        parent_bbox = self._get_bbox_to_anchor()
+        fontsize = renderer.points_to_pixels(self.get_size())
+        container = parent_bbox.padded(-0.5 * fontsize)
+
+        # use the same checking order as legend for determinism
+        coefs = ["NE", "NW", "SW", "SE", "E", "W", "E", "S", "N", "C"]
+        consider_bboxes = [bbox.anchored(c, container=container)
+                for c in coefs]
+        consider = [(b.x0, b.y0) for b in consider_bboxes]
+        pos = find_best_position(self.axes, bbox.width, bbox.height, consider)
+        self.set_position(pos)
+        self.anncoords = "axes pixels"
```

Comment:
this is a slightly weird/funny looking line of code lol, just setting the property to a string literal. I should look into this code more and learn what's happening here :P

Reply:
anncoords is a string, indicating the type of the coordinates (i.e. is it in data space, pixel space, etc.)